shared/actions/platform-specific/index.native.js

```diff
@@ -287,6 +293,16 @@ function* loadStartupDetails() {
   const initialShare = yield Saga._fork(getStartupDetailsFromShare)
   const [routeState, link, push, share] = yield Saga.join(routeStateTask, linkTask, initialPush, initialShare)
 
+  // Clear last value to be extra safe bad things don't hose us forever
+  yield Saga._fork(() => {
+    RPCTypes.configSetValueRpcPromise({
+      path: 'ui.routeState2',
+      value: {isNull: false, s: ''},
```

Comment:
This seems weird. What's the point of `isNull` if you're just passing an empty string?

Reply:
we don't care about this isNull capability of this api. we're fine with it just being a string